moped-database/migrations/1712697601392_add_full_project_name_column/up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE moped_project
+ADD project_name_full text GENERATED ALWAYS AS (COALESCE(project_name || ' - ' || project_name_secondary, project_name)) STORED;
```

Comment:
I tried using `concat_ws` here which would have been nice, but I found that it is not considered immutable in Postgres so it isn't a fit for generated columns. [Thread here with some details](https://www.postgresql.org/message-id/CA%2Bu7OA6AD9OLziFW%2Bio1HgN8q_XH0o1Y5RyufXYO5%3D0fnhG5zQ%40mail.gmail.com)